lib/linter-integration.js

```diff
@@ -31,13 +31,15 @@ class Linter {
     }
     this.linter.setMessages(messages.map(match => ({
       type: match.type || 'Error',
-      text: match.message || 'Error from build',
+      text: !match.message && !match.html_message ? 'Error from build' : match.message,
+      html: match.html_message,
```

Comment:
Yes, you're right. Here I saw two options: either we check it on our side and throw an error or let Linter make all the checks and throws errors. Now, other checks are made by Linter (for instance, that `path` is non-empty) so I decided to leave it this way. Although, if you think it must be the builder's responsibility to check the duplication, I'll add it. I think both attitudes have pros and cons and I have no strict preference other than being consistent in this question.